webui/react/package.json

```diff
@@ -103,51 +102,41 @@
     "@types/smoothscroll-polyfill": "^0.3.1",
     "@types/sprintf-js": "^1.1.2",
     "@types/throttle-debounce": "^5.0.0",
-    "@types/webpack": "^5.28.0",
     "@typescript-eslint/eslint-plugin": "^5.48.0",
     "@typescript-eslint/parser": "^5.48.0",
     "@vitejs/plugin-react-swc": "^3.1.0",
+    "@vitest/coverage-c8": "^0.28.5",
     "browserslist": "^4.21.4",
     "cheerio": "^1.0.0-rc.12",
-    "craco-sass-resources-loader": "^1.1.0",
-    "csp-html-webpack-plugin": "^5.1.0",
     "eslint": "^7.32.0",
     "eslint-config-prettier": "^8.6.0",
     "eslint-config-react": "^1.1.7",
     "eslint-import-resolver-typescript": "^2.7.1",
     "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-jest": "^24.7.0",
     "eslint-plugin-jsdoc": "^39.6.8",
     "eslint-plugin-react": "^7.31.11",
     "eslint-plugin-react-hooks": "^4.6.0",
     "eslint-plugin-sort-keys-fix": "^1.1.2",
     "express": "^4.17.1",
-    "jest-junit": "^13.0.0",
-    "less": "^4.1.1",
-    "less-loader": "^7.0.2",
+    "jsdom": "^16.7.0",
     "magic-string": "^0.27.0",
-    "monaco-editor-webpack-plugin": "^3.0.1",
     "morgan": "^1.10.0",
     "npm-force-resolutions": "0.0.10",
     "prettier": "^2.8.0",
-    "react-app-rewired": "^2.1.6",
-    "react-docgen-typescript-loader": "^3.7.2",
-    "react-scripts": "^4.0.3",
     "resize-observer-polyfill": "^1.5.1",
     "sass": "^1.35.2",
-    "sass-loader": "^10.1.1",
     "source-map-explorer": "^2.5.0",
     "stylelint": "^13.13.1",
     "stylelint-config-prettier": "^9.0.3",
     "stylelint-config-standard": "^22.0.0",
     "stylelint-order": "^4.1.0",
     "stylelint-scss": "^3.18.0",
-    "ts-node": "^10.9.1",
-    "ts-node-dev": "^2.0.0",
     "typescript": "^4.8.4",
     "vite": "^4.1.1",
     "vite-plugin-checker": "^0.5.5",
-    "vite-tsconfig-paths": "^4.0.5"
+    "vite-tsconfig-paths": "^4.0.5",
+    "vitest": "^0.28.5",
+    "whatwg-fetch": "^3.6.2"
```

Comment:
No -- we're only using `whatwg-fetch` for the `Response` and `Request` globals it provides.

Reply:
i tried it without it, and it worked. this can be removed If i did it correctly. it is required for jest tho

Reply:
@keita-determined Tried removing `whatwg-fetch` and the service utils module tests failed because they need `Response` to be global. Is it possible that you removed the package but not where we import it and it worked because we still have the package from `portable-fetch`?

Reply:
so i think i tested incorrectly. thanks for the clarification